tools/build.sh

```diff
@@ -47,9 +47,15 @@ echo ''
 if [ -z $2 ]; then
   echo "make sharedlib"
   make sharedlib
+  echo "=> shared library directory: ./model/configuration/"
 else
   echo "make sharedlib" $2
   make sharedlib SHAREDLIBDIR=$2
```

Comment:
`if [ -z  $2]` returns True if `$2` is empty. So if no argument is supplied, `make` runs with the default `$SHAREDLIBDIR` within `Makefile`. If an argument is supplied here, `make` uses that instead of its own `$SHAREDLIBDIR`. So I don't see any reason to change the location, and doing so would mean you can't use `make` directly, and would be forced to always use `build.sh`.

Reply:
Okay, sounds sensible to me. So `model/configuration` is default for all `mechanism.*` files as well as for `mechanism.so`. It is just that the places were the default values are defined are different (`mech_converter.py` and `Makefile`). Right?

Reply:
Yes. The two are independent, but just happen to have the same default location/.